sdk/python/kfp/_client.py

```diff
@@ -312,6 +312,79 @@ def run_pipeline(self, experiment_id, job_name, pipeline_package_path=None, para
     Returns:
       A run object. Most important field is id.
     """
+    job_config = self._create_job_config(
+      experiment_id=experiment_id,
+      params=params,
+      pipeline_package_path=pipeline_package_path,
+      pipeline_id=pipeline_id,
+      namespace=namespace)
+    run_body = kfp_server_api.models.ApiRun(
+        pipeline_spec=job_config.spec, resource_references=job_config.resource_references, name=job_name)
+
+    response = self._run_api.create_run(body=run_body)
+
+    if self._is_ipython():
+      import IPython
+      html = ('Run link <a href="%s/#/runs/details/%s" target="_blank" >here</a>'
+              % (self._get_url_prefix(), response.run.id))
+      IPython.display.display(IPython.display.HTML(html))
+    return response.run
+
+  def run_pipeline_recurring(self, experiment_id, job_name, start_time=None, end_time=None, interval_second=600 , max_concurrency=1, params={}, pipeline_package_path=None, pipeline_id=None, namespace=None, enabled=True):
```

Comment:
Should we rename this to `create_recurring_run` or  `create_recurring_job` to be in line with the API and `list_recurring_runs`?

Reply:
I renamed it `create_recurring_run`.